src/bundler.ts

```diff
@@ -79,7 +79,7 @@ class Bundler {
                     String(opts.abspath).trim() !== '') ?
         path.resolve(opts.abspath) :
         undefined;
-    this.pathResolver = new PathResolver(Boolean(this.abspath));
+    this.pathResolver = new PathResolver(this.abspath);
```

Comment:
I think we can get away with just a `prefix`, which could be applied by a final pass of the pathResolver after all inlining is completed. I think there's a value to the `basePath` argument, but I'm not sure that we've had any demand for that.

The concept of a `root` will have to exist as a command line argument, but nothing in the Bundler class needs to know about it, since the Analyzer will take care of it.